Project-Hotpot/Project-Hotpot/ViewControllers/HomeViewController.swift

```diff
@@ -10,12 +10,11 @@ import UIKit
 class HomeViewController: UIViewController {
     var api_instance = SpotifyManager.shared()
     @IBOutlet weak var card: UIView!
+    @IBOutlet weak var playButton: UIButton!
```

Comment:
Common to use ! here in controllers?

Reply:
Whenever I create an outlet from the storyboard to the controller / code, it automatically sets it like this.

Reply:
`@IBOutlet`s are a bit of a gray area, I will concede this 🙂 

I personally tend to make these optional as well; usually this doesn't cause significant extra `guard`s and `if let`s in the code (most uses are inline, e.g. `playButton?./* stuff */`). But I also understand the argument that these live in a storyboard and should always be populated, so I'm not going to be hard-over on these.